cxplat/inc/cxplat_common.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+// Include platform-specific definitions of common defines.
+#include "cxplat_platform.h"
+
+#include <assert.h>
```

Comment:
FYI, in MsQuic's platform abstraction, the definition of assert is platform specific, and does not use `assert.h`. On Windows we have:
```c
#define CXPLAT_ASSERT_LOG(_exp, _msg) \
    (CXPLAT_ANALYSIS_ASSUME(_exp), \
    ((!(_exp)) ? (CxPlatLogAssert(__FILE__, __LINE__, #_exp), FALSE) : TRUE))

#define CXPLAT_ASSERT_CRASH(_exp, _msg) \
    (CXPLAT_ANALYSIS_ASSUME(_exp), \
    ((!(_exp)) ? \
        (CxPlatLogAssert(__FILE__, __LINE__, #_exp), \
         __annotation(L"Debug", L"AssertFail", _msg), \
         DbgRaiseAssertionFailure(), FALSE) : \
        TRUE))
```